lib/ClangImporter/SwiftBridging/swift/bridging

```diff
@@ -164,6 +164,14 @@
 #define SWIFT_NONESCAPABLE \
   __attribute__((swift_attr("~Escapable")))
 
+/// Specifies that the return value is passed as owned for C++ functions and
+/// methods returning types annotated as SWIFT_SHARED_REFERENCE
```

Comment:
Minor: let's put macro names in backticks (``), to make sure it gets highlighted as code in IDE's quick help popup.

Reply:
Also I may change this  comment in a future patch to "for functions and methods returning" instead of for "C++ functions and methods returning". Because eventually we want these annotations to be supported for ObjC (and maybe other language's) methods and functions also which return a C++ foreign reference types (or SWIFT_SHARED_REFERENCE).